.github/ISSUE_TEMPLATE/PPR_TEMPLATE.md

```diff
@@ -0,0 +1,206 @@
+
+#### CI/CD
+- [ ] Are we producing any build artifacts ? (list artifact names)
+- [ ] What is the release schedule release ?
+- [ ] Is there a downtime involved in the release ? If yes, what is the expected downtime ?
+- [ ] Is there a rollback process,  how long does it take ?
+
+##### If the CI/CD is not using the current standard
+- [ ] Where are build artifacts stored and how long are they retained ?
+- [ ] Does CI process consist of following
+    - [ ] linting
+    - [ ] unit testing
+    - [ ] security testing
+    - [ ] code scanning (`sonarqube`)
+- [ ] How are build artifacts labelled ? How many copies of build artifact do we store ( `:latest`, `:semVer`) ?
+- [ ] Is release process fully automated ? If not, list how automated and non-automated steps.
+- [ ] What deployment strategy are we using ( `blue-green, canary, A/B, Recreated, Shadow`)
+
+### Database
+
+- [ ] Does this service use any database ?
+- [ ] Is database schema verified by Database Team ?
+- [ ] What is the expected growth rate ?
+
+### Patching
+
+- [ ] How often is the service patched ?
+- [ ] How are dependencies maintained ?
+- [ ] Is patching fully automated ? If not, list of the automated and non automated ones ?
+- [ ] Is there a runbook for patching of service, infrastructures and application used by this feature ?
+- [ ] Is there an expected downtime during patching ? If yes, how long it is expected ?
+
+
+### Security and Compliance
+
+##### Elevated access control
+- [ ] Are there users that have elevated access ?
+- [ ] If so, how are they managed ?
+- [ ] Is there an elevated access audit / approval process ?
+
+##### Design
+- [ ] Does service follow company infosec guidelines ?
+- [ ] Is anything not using HTTPS for communication ?
+- [ ] How are certificates managed ? Are they renewed manually or automatically ?
+
+##### Infrastructure
+- [ ] Are there any new resources added that this service uses ?
+- [ ] Is IaC(terraform) used to creat infrastructure required for this service ?
+    - [ ] If yes, is IaC code checked using secure code analysis tool ?
+    - [ ] Where is the state file stored  and who has access to it ?
+    - [ ] Is statefile backed up from remote storage ?
+    - [ ] Is statefile version controlled ?
+    - [ ] Is there any secret in the statefile ?
+- [ ] Are we using configuration management tool like ansible
+  - [ ] How are variables and secrets passed ?
+- [ ] Are there network security policy applied
+    - [ ] Do firewall follow the principle of least principle
+    - [ ] Is service able to handle and mitigate DDOS attack ?
+    - [ ] Is service behind IDS / IPS ?
+
+
+##### Development and Operations
+
+- [ ] Is security scanning done as part of pipeline ?
+- [ ] Are all the secrets encrypted at Rest and in Transit ?
+- [ ] Is this service adding any form of authentication and authorization ?
+- [ ] Does service follow principle of least previlege ?
+- [ ] Are there any audit logs for data access ?
+
+##### Logs
```

Comment:
maybe a point about data retention periods for logs?

Reply:
log retention is set by gov't, but maybe we need to mention it?